polkadotjs-examples/src/interfaces/augment-api-consts.ts

```diff
@@ -0,0 +1,116 @@
+// Auto-generated via `yarn polkadot-types-from-chain`, do not edit
```

Comment:
Are these files expected to be kept under version control? In #85 they are explicitly excluded via `.gitignore`.

Reply:
Since it is for examples it seems easier to not force the end user to generate the types. Also, this allows us more control when metadata changes.